SnaffCore/ActiveDirectory/AdData.cs

```diff
@@ -51,7 +51,25 @@ public List<string> GetDfsNamespacePaths()
         public DirectoryContext DirectoryContext { get; set; }
         private List<string> DomainControllers { get; set; } = new List<string>();
 
-        public AdData()
+        private string GetNetBiosDomainName(string dnsDomainName)
+        {
+            string ldapBase = $"CN=Partitions,CN=Configuration,DC={_targetDomain.Name.Replace(".", ",DC=")}";
+
+            DirectorySearch ds = new DirectorySearch(dnsDomainName, dnsDomainName, ldapBase, null, null, 0, false);
```

Comment:
should be able to do `new DirectorySearch(_targetDomain, _targetDC, ldapBase, null, null, 0, false)` if you make _targetDomain a string again instead of a Domain object.